### PR TITLE
Conference Banner and Participation Update

### PR DESCRIPTION
**Pull request by mrmundt**
## Description
This PR implements changes:

- Add the conference banner to the main page
- Add the submission information / dates to the main page
- Add the submission URL to the participate page

## Checklist:

- [ ] (Committee Chairs Only) I have posted the link for the PR in the usrse slack (#usrse-2024-conference-planning-committee) to ask for content reviewers
- [x] I have previewed changes locally
- [ ] I have updated the README.md if necessary

cc **ADD GITHUB HANDLES HERE**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,13 +25,34 @@ <h3>But what <i>is</i> an RSE?</h3>
...
+        <li>RSE impact</li>
+        <li>Efforts to expand the RSE movement</li>
+        <li>Research data management</li>
+        <li>Software engineering approaches supporting research</li>
+        <li>Diversity, Equity and Inclusion for RSEs and in RSEng</li>
+        <li>Training and workforce development</li>
+        <li>Maturing and expanding the RSE profession</li>
+      </ul>
+      
+      <p><b>Submission Website</b>: <a href="https://easychair.org/conferences/?conf=usrse2024" target="_blank">EasyChair</a></p>
```
Comment (danielskatz) [inline index.html:45]:
```suggestion
      <p><b>Submission Website</b>: <a href="https://easychair.org/conferences/?conf=usrse2024" target="_blank">EasyChair</a>; please see submission templates below for each type of submission</p>
```

File: pages/participate/participate.md [hunk 2]
```diff
@@ -25,6 +25,8 @@ include, but are not limited to:
 - Training and workforce development
 - Maturing and expanding the RSE profession
 
+**Submission Website**: [EasyChair](https://easychair.org/conferences/?conf=usrse2024)
```
Comment (danielskatz) [inline pages/participate/participate.md:28]:
```suggestion
**Submission Website**: [EasyChair](https://easychair.org/conferences/?conf=usrse2024); please see submission templates below for each type of submission
```

**Review by danielskatz — APPROVED** [review]
fine, though see my suggestion and addition question as well

File: pages/participate/participate.md [hunk 3]
```diff
@@ -25,8 +25,20 @@ include, but are not limited to:
 - Training and workforce development
 - Maturing and expanding the RSE profession
 
+**Submission Website**: [EasyChair](https://easychair.org/conferences/?conf=usrse2024);
+please see submission templates below for each type of submission.
+
+**Review and Publication**: All contributions will be subject to standard peer
```
Comment (danielskatz) [inline pages/participate/participate.md:31]:
```suggestion
**Review and Publication**: All contributions will be subject to non-anonymous peer
```

File: pages/participate/participate.md [hunk 4]
```diff
@@ -25,8 +25,20 @@ include, but are not limited to:
...
+review for quality, relevance, and novelty, which will determine acceptance and the form of
```
Comment (danielskatz) [inline pages/participate/participate.md:32]: the form of presentation isn't part of the review process - form is what was submitted.  The reviewers only say yes or no to if it is presented in that format.

File: pages/participate/participate.md [hunk 5]
```diff
@@ -25,8 +25,20 @@ include, but are not limited to:
...
+presentation. All submissions will be kept confidential prior to publication in
+the proceedings. Instructions for camera-ready versions will be sent to authors
+of accepted submissions. All submission types will be published as open access
+in Zenodo.
+
 Accepted abstracts will be placed on the website ahead of the conference.
 
+**Questions?** Contact the organizers: [usrse2024@easychair.org](mailto:usrse2024@easychair.org).
```
Comment (danielskatz) [inline pages/participate/participate.md:40]:
```suggestion
**Questions?** Contact the technical program organizers: [usrse2024@easychair.org](mailto:usrse2024@easychair.org).
```

**Comment by mrmundt** [comment]
Previews:
- [Home page](https://output.circle-artifacts.com/output/job/20b49875-286d-4a48-9994-b3b34d4f73a0/artifacts/0/usrse.github.io/usrse24/index.html)
- [Participate page](https://output.circle-artifacts.com/output/job/20b49875-286d-4a48-9994-b3b34d4f73a0/artifacts/0/usrse.github.io/usrse24/participate/index.html)

**Review by danielskatz — APPROVED** [review]

**Review by lmilechin — APPROVED** [review]
Looks good, love the banner!

**Pull request merged**